circle-test.sh

```diff
@@ -9,18 +9,12 @@ set -e
 DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
 cd $DIR
 
-export OUTPUT_DIR="$CIRCLE_ARTIFACTS" 
+export OUTPUT_DIR="$CIRCLE_ARTIFACTS"
```

Comment:
It's new for this branch.  It prevents circle from running tests because 1.3 uses 3 builders, but master is using 5.